main/test/ca/uwaterloo/flix/TestUtils.scala

```diff
@@ -72,9 +73,7 @@ trait TestUtils {
     */
   def expectSuccess(result: Validation[CompilationResult, CompilationMessage]): Unit = result match {
     case Validation.Success(_) => ()
-    case Validation.Failure(errors) =>
-      val actuals = errors.map(_.getClass)
```

Comment:
This variable was unused. Should it be used in below in the `fail` call?

Reply:
Just remove it, if it was unused.